main.tex

```diff
@@ -15,35 +15,33 @@
 \maketitle
 
 \begin{abstract}
+  Bitcoin's design is based on pseudonymous identities, and a publicly verifiable ledger of transactions transferring value between them. Although this design lacks strong guarantees for privacy, it does not rule it out.
+  Despite the potential for private use, research has shown that users' activity can often be traced in practice.
+  This lack of inherent privacy has given rise to business models built on dragnet surveillance of Bitcoin users which is problematic for Bitcoin's fungibility.
 
-% TODO citations should not be included in the abstract, the abstract should be cleaned up, and the remaining information and citations split off into an introduction section with citations
+  A number of methods have been proposed to mitigate these issues. Among these, CoinJoin is an approach to structuring transactions which adds ambiguity and breaks some common assumptions that underlie heuristics used for surveillance.
```

Comment:
```suggestion
  A number of methods have been proposed to mitigate these issues. Among these, CoinJoin is an approach to structuring transactions which adds ambiguity and breaks some assumptions those underlie heuristics used for surveillance.
```

Reply:
"those" is not grammatical in this context, what's the problem with that? also why remove common? i think it's relevant that they have been circulating widely and are commonly held

Reply:
> "those" is not grammatical in this context

ACK.

> also why remove "common"

How about removing the "some" instead?

Reply:
```suggestion
  A number of methods have been proposed to mitigate these issues. Among these, CoinJoin is an approach to structuring transactions which adds ambiguity and breaks common assumptions that underlie heuristics used for surveillance.
```